graphqlapi/filters.go

```diff
@@ -0,0 +1,175 @@
+package graphqlapi
+
+import (
+	"github.com/graphql-go/graphql"
+)
+
+type filterDescriptorsContainer struct {
+	filterNames             map[string]string
+	filterDescriptions      map[string]string
+	filterFetchNames        map[string]string
+	filterFetchDescriptions map[string]string
+}
+
+// TODO: store string values in constants
+// TODO: think of a better name for this struct
+func initializeFilterDescriptor() *filterDescriptorsContainer {
```

Comment:
You're doing what is described in the spec, but it feels a bit convoluted. I'd like to wait on a response by @bobvanluijt on https://github.com/creativesoftwarefdn/weaviate/issues/441.

(please don't resolve this comment).

Reply:
Not reviewing the filters until we have a answer on #411.